experiments/long_runs/leaderboard/leaderboard.jl

```diff
@@ -0,0 +1,165 @@
+
+        # Remove first spin_up_months months from simulation
+        spin_up_months = 12
+        spinup_cutoff = spin_up_months * 30 * 86400.0
+        ClimaAnalysis.times(sim_var)[end] >= spinup_cutoff && (
+            sim_var =
+                ClimaAnalysis.window(sim_var, "time", left = spinup_cutoff)
+        )
+
+        # Get 12 or less months of data
+        num_times = length(ClimaAnalysis.times(sim_var))
+        num_times > 12 && (
+            sim_var = ClimaAnalysis.window(
+                sim_var,
+                "time",
+                right = ClimaAnalysis.times(sim_var)[12],
+            )
+        )
+
+        obs_var = ClimaAnalysis.resampled_as(obs_var, sim_var)
+
+        sim_obs_comparsion_dict[short_name] = (sim_var, obs_var)
+    end
+
+    # Get the right number of months for plotting
+    _, sim_obs_tuple = first(sim_obs_comparsion_dict)
+    num_times = length(ClimaAnalysis.times(sim_obs_tuple[1]))
+    months = Dates.monthname.(1:num_times)
+
+    # Plot monthly comparsions
+    for short_name in short_names
+        fig = CairoMakie.Figure(size = (650 * ceil(num_times / 2), 450 * 2))
+        sim_var, obs_var = sim_obs_comparsion_dict[short_name]
+        mask = mask_dict[short_name](sim_var, obs_var)
+        times = ClimaAnalysis.times(sim_var) |> copy
+        times = vcat(
+            times,
+            Array{Union{Missing, eltype(times)}}(missing, 12 - num_times),
+        )
+        times = reshape(times, (2, 6))
+        for ((indices, t), month) in zip(pairs(times), months)
+            layout = fig[Tuple(indices)...] = CairoMakie.GridLayout()
+            ClimaAnalysis.Visualize.plot_bias_on_globe!(
+                layout,
+                ClimaAnalysis.slice(sim_var, time = t),
+                ClimaAnalysis.slice(obs_var, time = t),
+                cmap_extrema = compare_vars_biases_plot_extrema[short_name],
+                mask = mask,
+            )
+            CairoMakie.Label(
+                layout[0, 1],
+                month,
+                tellwidth = false,
+                fontsize = 30,
+            )
+        end
+        CairoMakie.save(
+            joinpath(leaderboard_base_path, "$(short_name)_bias_plot.png"),
+            fig,
+        )
+    end
+
+    # Plot month (x-axis) and global bias and global RMSE (y-axis)
+    fig = CairoMakie.Figure(size = (450 * length(keys(sim_var_dict)), 600))
+    for (col, short_name) in enumerate(keys(sim_var_dict))
+        sim_var, obs_var = sim_obs_comparsion_dict[short_name]
+        mask = mask_dict[short_name](sim_var, obs_var)
+        # Compute globas bias and global rmse
+        rmse_vec = []
+        bias_vec = []
+        times = ClimaAnalysis.times(sim_var) |> copy
+        for t in times
+            g_rmse = ClimaAnalysis.global_rmse(
+                ClimaAnalysis.slice(sim_var, time = t),
+                ClimaAnalysis.slice(obs_var, time = t),
+                mask = mask,
```

Comment:
ClimaAnalysis uses the mask to change the denominator in RMSE and mean bias?

Reply:
Yes, the mask is used to normalize the global RMSE and global bias.